queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE animal_name LIKE '%mon%';
```

Comment:
Please write a query to find all animals that have names ending with "mon", not with names starting and ending with "mon".

Reply:
Thanks, I'll work on it!